### DISCUSSION TITLE
srsran project with different Near real time RIC

### DISCUSSION THREAD

**Discussion by Jayanthhn14** [srsran/srsRAN_Project#391] [General Help]
Hello,

I'm new to this and exploring the srsran project, where flexric is used as the near real-time RIC. I'm curious if it's possible to use VMware as the near real-time RIC and how to go about implementing it. Any guidance would be greatly appreciated.

Thank you!

**Comment by andrepuschmann** [comment]
So far we've not gotten our hands on a VMware RIC but it's something we are working on actively.

**Comment by pgawlowicz** [comment]
@Jayanthhn14, you might be interested in our new tutorial on how to use our E2 agent with ORAN-SC-RIC. 
Please check the `Quick Start` section here: https://github.com/srsran/oran-sc-ric
